shared/mad/codeql/mad/modelgenerator/ModelGeneratorImpl.qll

```diff
@@ -0,0 +1,909 @@
+/**
+ * INTERNAL: Do not use.
```

Comment:
If that's the case, then better to put this file inside an `internal` folder.

Reply:
Good idea :-D